test/testdata/lsp/definitions_and_usages.rb

```diff
@@ -19,14 +19,16 @@ def method(a, b, &block)
 
   def method_with_keyword_args(
     meaning_of_life: 42,
-  # ^^^^^^^^^^^^^^^ def: meaning_of_life
+  # ^^^^^^^^^^^^^^^ def: meaning_of_life 1
+                   # ^^ def: meaning_of_life 2 default-arg-value
```

Comment:
What happens if this annotation weren't present?

Reply:
You get an error about sorbet not reporting a reference to `meaning_of_life`:
```
test/testdata/lsp/definitions_and_usages.rb:21: ERROR: Sorbet did not report a reference to symbol `meaning_of_life`.
Given symbol at:
21:     meaning_of_life: 42,
        ^
Sorbet did not report reference at:
21:     meaning_of_life: 42,
                         ^^
```

Reply:
gotcha. i thought this was the point that `1,2` syntax was trying to solve?

Reply:
They're related -- the `1,2` syntax was needed to depend on the two different definitions of the variable, and `default-arg-value` was needed to avoid some checks on the `def` that corresponds to the default value.